packages/sdk/src/controllers/ConnectorController.ts

```diff
@@ -189,6 +189,47 @@ export class ConnectorController {
         return res.getConnectorOptions(id).then((result) => getEditorResponseData<ConnectorOptions>(result));
     };
 
+    /**
+     * Sets the data source by id.
+     * @param connectorId the id of your data connector
+     * @returns
+     */
+    setDataSource = async (connectorId: string) => {

```

Comment:
I'm not sure that having such methods in ConnectorController is a good idea. First of all because it's related to data Connector only, where this controller created to work with any type of connector. In addition, if I'm not mistaken, there where discussion to have it as new controller, called `dataSource`

Reply:
Fine with a data source controller tbh.

Logic of avoiding the data controller (which I considered first) is that they are all those type specific connector controllers are related to direct interaction with the running controller (js) maybe that's me thinking too much with implementation in mind and does not matter so much for the user.

I think data source controller is probably less confusing anyway.